From 9ef42b78435ca3c593d7589a8844886d735a646b Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Thu, 5 Oct 2017 14:17:05 -0400
Subject: wdmd Kernel watchdog enable and reboot on error

This patch is a port of this commit:

5a4deca10fd8e967c042714382dd66bf9a4cf15c
(CGTS-3360 - Kernel watchdog enable and reboot on error)

into centos 7.

---
 init.d/wdmd.service                             |  4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.d/wdmd.service b/init.d/wdmd.service
index 7e6d973..2e7db50 100644
--- a/init.d/wdmd.service
+++ b/init.d/wdmd.service
@@ -3,9 +3,9 @@ Description=Watchdog Multiplexing Daemon
 After=syslog.target
 
 [Service]
-Type=forking
-ExecStart=/lib/systemd/systemd-wdmd start
+Type=oneshot
 ExecStop=/lib/systemd/systemd-wdmd stop
+RemainAfterExit=yes
 
 [Install]
 WantedBy=multi-user.target
-- 
1.9.1

